samtranslator/translator/managed_policy_translator.py

```diff
@@ -11,6 +13,8 @@ def __init__(self, iam_client):
         self._policy_map = None
         self.max_items = 1000
 
+        self.iam_metrics = Metrics(namespace="Downstream-retry", metrics_publisher=EMFMetricsPublisher())
```

Comment:
Should we hardcode `Downstream-retry` or should we pull this from an env var we set on the service?